HeliosFalcon/Interfaces/FalconInterface.cs

```diff
@@ -260,7 +260,10 @@ public string[] GetFalconVersions()
             if(Registry.LocalMachine.OpenSubKey(falconRootKey) != null)
             {
                 subkeys = Registry.LocalMachine.OpenSubKey(falconRootKey).GetSubKeyNames();
-                Array.Reverse(subkeys);
+                if (subkeys.Length > 0)
```

Comment:
if you use subkeys.SubKeyCount instead of subkeys.Length you'll get access to other exceptions that would spell out what the issue is. SubKeyCount will throw on these exceptions:

Exceptions
[SecurityException](https://learn.microsoft.com/en-us/dotnet/api/system.security.securityexception?view=net-7.0)
The user does not have read permission for the key.
[ObjectDisposedException](https://learn.microsoft.com/en-us/dotnet/api/system.objectdisposedexception?view=net-7.0)
The [RegistryKey](https://learn.microsoft.com/en-us/dotnet/api/microsoft.win32.registrykey?view=net-7.0) being manipulated is closed (closed keys cannot be accessed).
[UnauthorizedAccessException](https://learn.microsoft.com/en-us/dotnet/api/system.unauthorizedaccessexception?view=net-7.0)
The user does not have the necessary registry rights.
[IOException](https://learn.microsoft.com/en-us/dotnet/api/system.io.ioexception?view=net-7.0)
A system error occurred, for example the current key has been deleted.

Reply:
Just been testing the Falcon Interface with a registry key having no subkeys, there are some more instances where an exception is thrown. I'll revise the PR as soon as I get some time.